test/framework/kubernetes.go

```diff
@@ -218,3 +218,15 @@ func CreateSecret(namespace, name string, secretContent map[string]string) error
 
 	return kubernetes.ResourceC(kubeClient).Create(secret)
 }
+
+// CheckPodHasImagePullSecretWithPrefix checks that a pod has an image pull secret starting with the given prefix
+func CheckPodHasImagePullSecretWithPrefix(pod *corev1.Pod, imagePullSecretPrefix string) bool {
+	if len(pod.Spec.ImagePullSecrets) > 0 {
```

Comment:
Just a note that this is unnecessary even if ImagePullSecrets is nil.

Reply:
oh right ! will remove that